src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
```

Comment:
TodoItem will be a better name. When i saw 'Todo' i was thinking more about whole Todo implementation